mesos/cli/parser.py

```diff
@@ -51,3 +51,9 @@ def path_argument(self):
             'path', type=str, nargs="?", default="",
             help="""Path to view."""
         ).completer = completion_helpers.file
+
+    def enable_command_arguments(self, help_msg="(Optional) Any options to pass through to the system command."):
```

Comment:
`enable_command_arguments` -> `command_arguments` for consistency with the rest of the arguments.